### PR TITLE
Added the credentials tab

### PR DESCRIPTION
**Pull request by edewit**
## Motivation
Added credentials tab [marvel](https://marvelapp.com/prototype/695h94c/screen/72265889)

## Verification Steps

1. Go to client details
1. Make the client "public"
1. The new credentials tab will appear
-->

## Checklist:

- [x] Code has been tested locally by PR requester
- [x] User-visible strings are using the react-i18next framework (useTranslation)
- [x] Help has been implemented
- [ ] axe report has been run and resulting a11y issues have been resolved
- [ ] Unit tests have been created/updated
- [x] Formatting has been performed via prettier/eslint
- [x] Type checking has been performed via 'yarn check-types'

## Additional Notes
Setting the client to see the credentials tab:
image

### CODE REVIEW

File: src/clients/messages.json [hunk 1]
```diff
@@ -46,6 +48,20 @@
     "validRedirectUri": "Valid redirect URIs",
     "loginTheme": "Login theme",
     "consentRequired": "Consent required",
+    "clientAuthenticator": "Client Authenticator",
+    "clientSecret": "Client secret",
+    "regenerate": "Regenerate",
+    "confirmClientSecretTitle": "Regenerate secret for this client?",
+    "confirmClientSecretBody": "If you regenerate secret, the Keycloak database will be updated and you will need to download a new adapter for this client.",
+    "confirmAccessTokenTitle": "Regenerate registration access token?",
+    "confirmAccessTokenBody": "If you regenerate reiteration access token, the access data regarding the client registration service will be updated.",
```
Comment (srambach) [inline src/clients/messages.json:57]: Is it "reiteration" or "registration" ?

**Comment by srambach** [comment]
When there is an empty footer, it creates too much space below the Save button. You can have multiple `CardBody`s so maybe adding one as needed with the client secret would be a better way?

**Review by ssilvert — CHANGES_REQUESTED** [review]
"Client secret" missing.  To reproduce:
1. Create a new client
2. Go to Credentials tab
3. Default value for Client Authenticator is "Client Id and Secret"
4. "Client Secret" is missing

If I change "Client Authenticator" to something else, then change it back, "Client Secret" shows up.

**Comment by ssilvert** [comment]
`Client secret` has wrong help text.  You can leave this out for now since it isn't available in old console either.

**Comment by ssilvert** [comment]
For `Signed Jwt`, there is a switch and two buttons that exist in the old console, but not in the wireframes.  Please double-check to make sure this is OK.

**Comment by ssilvert** [comment]
`Save` button is always disabled whenever `Client Id and Secret` is selected.

**Comment by ssilvert** [comment]
When you save something, there is no alert.

**Comment by ssilvert** [comment]
On old console, `Signed Jwt with Client Secret` includes dropdown for `Signature Algorithm`

File: src/clients/messages.json (hunk 1 above)
Comment (edewit) [reply src/clients/messages.json:57]: I copied it from the old console, but I'm sure it should be registration.

**Comment by edewit** [comment]
> For `Signed Jwt`, there is a switch and two buttons that exist in the old console, but not in the wireframes. Please double-check to make sure this is OK.

According to the new design these settings will be on the new "keys" tab: 
image

**Comment by ssilvert** [comment]
> On old console, `Signed Jwt with Client Secret` includes dropdown for `Signature Algorithm`

Don't see a fix for this one?

**Comment by edewit** [comment]
> > On old console, `Signed Jwt with Client Secret` includes dropdown for `Signature Algorithm`
> 
> Don't see a fix for this one?

No, see my reply above! According to the new design these fields have moved to the new "Keys" tab.

**Review by ssilvert — APPROVED** [review]

**Pull request merged**